tests/testapp/settings.py

```diff
@@ -15,6 +15,7 @@
     "form_designer",
 ]
 
+DEBUG = True
```

Comment:
Let's continue running tests with `DEBUG=False` please.

Reply:
sorry, this is another side effect of my "run it locally" mode...so it's probably better to have a separate settings_dev.py